src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -448,6 +484,7 @@ public IntervalList getLociToGenotype(final Collection<Fingerprint> fingerprints
         // Now go through the data at each locus and figure stuff out!
         for (final SamLocusIterator.LocusInfo info : iterator) {
 
+            log.debug("At locus " + info.toString());
```

Comment:
This will do a pointless string combine at each position which could be kind of expensive if you don't want this output.  If there's a way check if we're in debug mode first than I would test that before doing the debug command.